topology/University of California San Diego/San Diego Supercomputer Center/SDSC-PRP.yaml

```
@@ -44,4 +44,29 @@ Resources:
         Description: GridFtp Storage Element
         Details:
           hidden: false
-
+  SDSC-PRP-OSPool-Provisioner:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Igor Sfiligoi
+          ID: 593daf4f680b3135849201036e1055f03f89cfac
+        Secondary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+        Tertiary:
+          ID: 38cd7e4efcb45e2aff808b98f5f928c96b3a8608
+          Name: Mats Rynge
+      Security Contact:
+        Primary:
+          Name: Igor Sfiligoi
+          ID: 593daf4f680b3135849201036e1055f03f89cfac
+        Secondary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+    Description: Direct provisioning of the PRP Kubernetes cluster for OSPool, in the osg-opportunistic namespace
+    FQDN: pacificresearchplatform.org
```

```suggestion
    FQDN: ospool-backfill.pacificresearchplatform.org
```

Just a minor suggestion to change this to something bogus since it seems like pacificresearchplatform.org resolves to a web server

---

Changed to
direct-provisioner.osg-opportunistic.pacificresearchplatform.org